src/main/java/com/cburch/logisim/fpga/hdlgenerator/Hdl.java

```diff
@@ -96,6 +116,41 @@ public static String assignOperator() {
     return isVhdl() ? " <= " : " = ";
   }
 
+  public static String equalOperator() {
+    return isVhdl() ? " = " : "==";
+  }
+
+  public static String notEqualOperator() {
+    return isVhdl() ? " \\= " : "!=";
+  }
+
+  private static String typecast(String signal, boolean signed) {
+    return isVhdl()
+                ? LineBuffer.formatHdl("{{1}}({{2}})", signed ? "signed" : "unsigned", signal)
```

Comment:
Please put `signed ? "signed" : "unsigned"` in brackets -> `(signed ? "signed" : "unsigned")` for better readability.